utils/errors/errors.go

```diff
@@ -163,8 +178,8 @@ func WrapWithSkip(err error, msg string, skip int) ErrorExt {
 	return WrapWithSkipAndStatus(err, msg, skip+1, nil)
 }
 
-//WrapWithSkip wraps an existing error and appends stack information if it does not exists skipping the number of function on the stack along with GRPC status
-func WrapWithSkipAndStatus(err error, msg string, skip int, status *grpcstatus.Status) ErrorExt {
+//WrapWithSkipAndStatus wraps an existing error and appends stack information if it does not exists skipping the number of function on the stack along with GRPC status
```

Comment:
Update the godoc comment to describe the new rawdata argument.

Reply:
Actually, based on a previous comment:

> 
> 
> why create new func only for raw data instead of adding for each Wrap* functions? rawData is accepting all types of args, what if we need to add new args as interface....for example,
> 
> ```
> func Wrap(err error, msg string, rawData ...interface{}) ..
> // become
> func Wrap(err error, msg string, anything interface{}, rawData ...interface{})
> ```
> 
> in case of it, I would prefer create new function to prevent potential breaking change in the future.

This has the same issue of potentially introducing a breaking change in the future.

Reply:
that's sort of true...I was thinking more about the purpose of function, so if the function indicating `RawData` it also implies you need to take the potential effort in the future for breaking change. Recently I have an idea on Notify func, will do the proposal soon to help the changes in this PR better.